code/addons/test/src/node/vitest-manager.ts

```diff
@@ -21,6 +23,7 @@ export class VitestManager {
   async startVitest(watchMode = false) {
     const { createVitest } = await import('vitest/node');
 
+    this.watchMode = watchMode;
```

Comment:
I think this is not necessary anymore?

Reply:
the cache is only cleared if watchmode is false, so it should be necessary